egs/callhome_diarization/v1/diarization/VB_resegmentation.sh

```diff
@@ -0,0 +1,103 @@
+  echo "  --alphaQInit <float|100.0>                       # Dirichlet concentraion"
+  echo "                                                   # parameter for initializing q"
+  echo "  --sparsityThr <float|0.001>                      # Set occupations smaller that"
+  echo "                                                   # this threshold to 0.0 (saves"
+  echo "                                                   # memory as the posteriors are"
+  echo "                                                   # represented by sparse matrix)"
+  echo "  --epsilon <float|1e-6>                           # Stop iterating, if obj. fun." 
+  echo "                                                   # improvement is less than" 
+  echo "				                   # epsilon"
+  echo "  --minDur <n|1>                                   # Minimum number of frames"
+  echo "                                                   # between speaker turns imposed"
+  echo "                                                   # by linear chains of HMM" 
+  echo "                                                   # state corresponding to each" 
+  echo "                                                   # speaker. All the states in"
+  echo "                                                   # a chain share the same output"
+  echo "                                                   # distribution"
+  echo "  --loopProb <float|0.9>                           # Probability of not switching"
+  echo "                                                   # speakers between frames"
+  echo "  --statScale <float|0.2>                          # Scale sufficient statistics" 
+  echo "                                                   # collected using UBM"
+  echo "  --llScale <float|1.0>                            # Scale UBM likelihood (i.e."
+  echo "                                                   # llScale < 1.0 make" 
+  echo "                                                   # attribution of frames to UBM"
+  echo "                                                   # componets more uncertain)" 
+  echo "  --channel <n|0>                                  # Channel information in the rttm file"
+  echo "  --initialize <n|1>                               # Whether to initalize the"
+  echo "                                                   # speaker posterior (if not)"
+  echo "                                                   # the speaker posterior will be"
+  echo "                                                   # randomly initilized"
+
+  exit 1;
+fi
+
+data_dir=$1
+init_rttm_filename=$2
+output_dir=$3
+dubm_model=$4
+ie_model=$5
+
+mkdir -p $output_dir/rttm
+
+sdata=$data_dir/split$nj;
+utils/split_data.sh $data_dir $nj || exit 1;
+
+if [ $stage -le 0 ]; then
+    $cmd JOB=1:$nj $output_dir/log/VB_resegmentation.JOB.log \
+      diarization/VB_resegmentation.py --true-rttm-filename $true_rttm_filename --max-speakers $max_speakers \
```

Comment:
`VB_resegmentation.py` doesn't support the `--true-rttm-filename` argument, are you committing the right versions?